frontend/src/queries/schema/schema-general.ts

```diff
@@ -1239,6 +1241,8 @@ export interface FunnelsQueryResponse
         FunnelStepsResults | FunnelStepsBreakdownResults | FunnelTimeToConvertResults | FunnelTrendsResults
     > {
     isUdf?: boolean
+    /** The date range used for the query */
+    query_date_range?: QueryDateRangeResponse
```

Comment:
This seems to be the query data range we pass as a payload along with the inside? Thus we should already have access to it frontend-side and shouldn't add it to the response, to stay consistent in how we handle displaying result items that depending on the query configuration.